.circleci/config.yml

```diff
@@ -154,6 +154,12 @@ jobs:
       - add_ssh_keys:
           fingerprints:
             - "53:1d:b4:b6:2f:4e:48:66:8e:eb:3d:67:bf:9e:d7:0d"
+      - run:
+          command: |
+            apt-get update
+            apt-get install -y npm
+            curl -o- -L https://yarnpkg.com/install.sh | bash
+          name: Install Yarn for Release Notes
```

Comment:
`yarn` should already be installed in the `ns8-php` image (but `gren` isn't).

Reply:
Yarn is not available in the image. I thought so too but had to SSH in and confirm so

Reply:
That can't be right... this repo is using the same image as protect-sdk-php which [does have yarn](https://github.com/ns8inc/protect-sdk-php/blob/master/.circleci/config.yml#L47).
```
MacBook-Pro:~ davehensley$ docker run -t -d 244249143763.dkr.ecr.us-west-2.amazonaws.com/ns8-php
c6842efe7347c4e176c694b85ce08299a052eabcef171050c0cfcae923b6553b
MacBook-Pro:~ davehensley$ docker exec -it c68 bash
root@c6842efe7347:/# which yarn
/usr/local/bin/yarn
root@c6842efe7347:/# 
```

Reply:
...I swear it didn't work before... It failed out with NPM previously. Either way, I'll re-try now

Reply:
If it would make things easier, I could update the Docker image to include a preinstalled copy of `gren` in the PATH so neither `protect-integration-magento` nor `protect-sdk-php` would need to install it before publishing.